extensions/positron-python/src/client/pythonEnvironments/common/externalDependencies.ts

```diff
@@ -139,27 +139,6 @@ export async function resolveSymbolicLink(absPath: string, stats?: fsapi.Stats,
     return absPath;
 }
 
-// --- Start Positron ---
-export function resolveSymbolicLinkSync(absPath: string, stats?: fsapi.Stats, count?: number): string {
-    stats = stats ?? fsapi.statSync(absPath);
```

Comment:
This had a bug :( related to https://stackoverflow.com/a/43443328

It was always returning `false` for everything. But since the code at the bottom of this PR is the only place we used this, and I switched that to async, we can now use the async `resolveSymbolicLink()` in this file.